hatchet_sdk/clients/dispatcher/action_listener.py

```diff
@@ -61,7 +61,7 @@ class Action:
     worker_id: str
     tenant_id: str
     workflow_run_id: str
-    get_group_key_run_id: Optional[str]
+    get_group_key_run_id: str
```

Comment:
I'm pretty sure this is supposed to be a string - lmk if that's wrong!

Reply:
It's only sent over the API when `hatchet.concurrency` functions are invoked...but I'm not sure if it's getting sent as an empty string or None.

Reply:
okay yeah just checked, it looks like it's an empty string!